rpc/core/blocks.go

```diff
@@ -140,7 +140,7 @@ func BlockByHash(ctx *rpctypes.Context, hash []byte) (*ctypes.ResultBlock, error
 
 // Commit gets block commit at a given height.
 // If no height is provided, it will fetch the commit for the latest block.
-// More: https://docs.tendermint.com/v0.37/rpc/#/Info/commit
+// More: https://docs.cometbft.com/main/rpc/#/Info/commit
```

Comment:
Why main and not 0.37?
```suggestion
// More: https://docs.cometbft.com/v0.37/rpc/#/Info/commit
```

Reply:
Thanks for noticing.